src/hotspot/share/gc/shared/genCollectedHeap.cpp

```diff
@@ -1069,16 +1069,3 @@ void GenCollectedHeap::record_gen_tops_before_GC() {
   }
 }
 #endif  // not PRODUCT
-
-class GenEnsureParsabilityClosure: public GenCollectedHeap::GenClosure {
- public:
-  void do_generation(Generation* gen) {
-    gen->ensure_parsability();
```

Comment:
This code should either be kept or the `Generation`s `ensure_parsability` also removed. Otherwise this looks like an overlooked forwarding to `Generation` after removal of this code.